### ISSUE TITLE
Suggestion: Add uMatrix style dynamic filtering syntax

### ISSUE THREAD

**Issue by joahim51** [gorhill/uBlock#2359]
I'm a firefox user, with uBlock0 & uMatrix addons, mostly to disable tracking, redundant scripts and other excessive stuff. I don't use Noscript as it's too complicated for me.

I have h264/webm/mse enabled in firefox, hardware acceleration enabled, i don't use npapi plugins, so no flash.

For instance when I go to Bloomberg there's video block on landing page which doesn't load until it loads a script from imasdk.googleapis.com which unfortunately was blocked few days ago in EasyList:
https://imasdk.googleapis.com/js/sdkloader/ima3.js
 Static filter ||imasdk.googleapis.com^$third-party found in: EasyList

Now they added to whitelist some of domains which use it:
Static filter @@||imasdk.googleapis.com/js/sdkloader/ima3.js$domain=allcatvideos.com|audiomack.com|bloomberg.com|cbc.ca|cnet.com|complex.com|gamejolt.com|news.sky.com|play.ludigames.com|player.performgroup.com|rumble.com|snopes.com|theverge.com|video.foxbusiness.com|video.foxnews.com|vidyomani.com found in: EasyList

Back to when it was blocked, I had to add following dynamic rule to uBlock0:
bloomberg.com https://imasdk.googleapis.com script allow
This was done only for the webpage to get access to bloomberg.tv (as it appeared in uMatrix only after applying that rule in uBlock0), without access to bloomberg.tv video didn't load.

To put it in a nutshell, imagine other situation when Easylist or other adblocking rule maintainers do not promptly add the needed site for me, and the link for the needed script will be constantly changing like cloudfront: d1z2jf7jlzjs58.cloudfront.net
It seems to me it would be easier to add:
somesite.com cloudfront.net script allow
Than using construction that constantly changes upon each reload or other circumstances including https:// or http://

So my suggestion is to add uMatrix style dynamic filtering syntax to uBlock0 to ommit such discrepancies.

Thanks.

**Comment by gorhill** [comment]
Duplicate of #1038.
